pkgs/applications/audio/sisco.lv2/default.nix

```diff
@@ -1,33 +1,35 @@
-{ stdenv, fetchurl, lv2, pkgconfig, mesa, cairo, pango, libjack2 }:
+{ stdenv, fetchFromGitHub, lv2, pkgconfig, mesa, cairo, pango, libjack2 }:
 
 let
-  name = "sisco.lv2-${version}";
+  name = "sisco.lv2-${src.rev}";
```

Comment:
the previous change was correct. The version shouldn't be prefixed with "v", it breaks the update detection otherwise.